.gitattributes

```diff
@@ -1,6 +1,6 @@
 # Handle line endings automatically for files detected as text
 # and leave all files detected as binary untouched.
-* text=auto
+* text=lf
```

Comment:
Can we back this out and open a separate PR for it if you truly want to change it?